### PR TITLE
feat: mobile interactions with input actions

### PR DESCRIPTION
**Pull request by kuruk-mm**
Fix https://github.com/decentraland/godot-explorer/issues/149

- is_mobile moved to Rust DclGlobal.rs, and set from Global.gd
- Fix interactions bugs
- Add icons, and effect to pressing a interaction

### CODE REVIEW

File: godot/src/ui/components/pointer_tooltip/pointer_tooltip.gd [hunk 1]
```diff
@@ -13,13 +13,17 @@ func set_pointer_data(interacts_array: Array):
 		child.queue_free()
 	var i = 0
 	for interact in interacts_array:
+		if i >= angles.size():
```
Comment (leanmendoza) [inline godot/src/ui/components/pointer_tooltip/pointer_tooltip.gd:16]: Does this discard the remaining interactions?

File: godot/src/global.gd [hunk 2]
```diff
@@ -30,11 +30,13 @@ var nft_frame_loader: NftFrameStyleLoader
 var standalone = false
 var dcl_android_plugin
 
-@onready var is_mobile = OS.has_feature("mobile")
-#@onready var is_mobile = true
-
 
 func _ready():
+	# First
+	_set_is_mobile(OS.has_feature("mobile"))
```
Comment (leanmendoza) [inline godot/src/global.gd:36]: could this be in the `init` function in Rust?

File: rust/decentraland-godot-lib/src/godot_classes/dcl_global.rs [hunk 3]
```diff
@@ -34,11 +34,14 @@ mod android {
     }
 }
 
+const IS_MOBILE: bool = cfg!(target_os = "android");
```
Comment (leanmendoza) [inline rust/decentraland-godot-lib/src/godot_classes/dcl_global.rs:37]: this won't be true for VR haha

**Review by leanmendoza — APPROVED** [review]
LGTM, just some comments.

File: rust/decentraland-godot-lib/src/godot_classes/dcl_global.rs (hunk 3 above)
Comment (kuruk-mm) [reply rust/decentraland-godot-lib/src/godot_classes/dcl_global.rs:37]: Haha, we can take care on that moment

File: godot/src/ui/components/pointer_tooltip/pointer_tooltip.gd (hunk 1 above)
Comment (kuruk-mm) [reply godot/src/ui/components/pointer_tooltip/pointer_tooltip.gd:16]: Yes. It avoids a crash. In the Foundation Client is crashing. And Godot was crashing too due to not having enough angles.
There is no limit to how many tooltip interactions you can have. 5 it's more than ok.

**Pull request merged**